<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashboardWindow</class>
 <widget class="QWidget" name="DashboardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dashboard - Apex Trust Bank</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
        /* Main Background */
        QWidget#DashboardWindow { background-color: #f0f4f8; }
        QFrame#headerFrame { background-color: white; border-bottom: 1px solid #e0e0e0; }
        QLabel#logoLabel { color: #0c3b6d; font-size: 22px; font-weight: bold; }
        QLabel#welcomeLabel { color: #333; font-size: 16px; font-weight: bold; }
        QPushButton#logoutBtn { background-color: #0c3b6d; color: white; border: none; border-radius: 4px; padding: 8px 15px; font-weight: bold; }
        QPushButton#logoutBtn:hover { background-color: #0a2e55; }
        QFrame#overviewCard { background-color: white; border: 1px solid #dcdcdc; border-radius: 6px; border-top: 4px solid #28a745; }
        QLabel#balanceTitle { color: #555; font-size: 14px; }
        QLabel#balanceValue_1 { color: #28a745; font-size: 36px; font-weight: bold; }
        QFrame#actionsCard { background-color: white; border: 1px solid #dcdcdc; border-radius: 6px; }
        QPushButton.actionBtn { color: white; border-radius: 8px; font-size: 14px; font-weight: bold; padding: 10px; }
        QPushButton#btnWithdraw { background-color: #0056b3; }
        QPushButton#btnDeposit { background-color: #28a745; }
        QPushButton#btnTransfer { background-color: #5bc0de; }
        QPushButton#btnModify { background-color: #1d2d44; }
        QPushButton.actionBtn:hover { opacity: 0.9; }
        QLabel#transTitle { color: #333; font-size: 18px; font-weight: bold; margin-bottom: 5px; }
        QTableWidget { border: 1px solid #dcdcdc; background-color: white; gridline-color: #f0f0f0; font-size: 14px; border-radius: 6px; }
        QHeaderView::section { background-color: #f8f9fa; border: none; border-bottom: 2px solid #0c3b6d; color: #333; font-weight: bold; padding: 8px; }
        QTableWidget::item { padding: 5px; }
        QFrame#footerFrame { background-color: #0c3b6d; }
        QLabel#footerLinks { color: white; }
      </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="headerFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QLabel" name="logoLabel">
        <property name="text">
         <string>Apex Trust Bank</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="welcomeLabel">
        <property name="text">
         <string>Welcome, User</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="logoutBtn">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Log Out</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1010</width>
        <height>835</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>40</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>40</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="QFrame" name="overviewCard">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>120</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_Balance">
          <property name="leftMargin">
           <number>30</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="balanceTitle">
            <property name="text">
             <string>Total Available Balance</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="balanceValue_1">
            <property name="text">
             <string>$0.00</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="actionsCard">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_Actions">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnWithdraw">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>üí∞ WITHDRAW</string>
            </property>
            <property name="class" stdset="0">
             <string>actionBtn</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDeposit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>‚¨áÔ∏è DEPOSIT</string>
            </property>
            <property name="class" stdset="0">
             <string>actionBtn</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnTransfer">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>‚áÜ TRANSFER</string>
            </property>
            <property name="class" stdset="0">
             <string>actionBtn</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnModify">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>‚öôÔ∏è MODIFY</string>
            </property>
            <property name="class" stdset="0">
             <string>actionBtn</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_filters">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QDateEdit" name="fromDateEdit">
           <property name="displayFormat">
            <string>yyyy-MM-dd</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="toDateEdit">
           <property name="displayFormat">
            <string>yyyy-MM-dd</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="typeCombo"/>
         </item>
         <item>
          <widget class="QPushButton" name="applyFilterBtn">
           <property name="text">
            <string>Apply Filters</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportCsvBtn">
           <property name="text">
            <string>Export CSV</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="transTitle">
         <property name="text">
          <string>üî¥ Live Recent Transactions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>350</height>
          </size>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <column>
          <property name="text">
           <string>Date</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Amount</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_End">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="footerFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_Footer">
      <item>
       <widget class="QLabel" name="footerLinks">
        <property name="text">
         <string>Privacy Policy | Contact Us</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
